x-pack/legacy/plugins/integrations_manager/public/constants.ts

```diff
@@ -6,39 +6,34 @@
 import { IconType } from '@elastic/eui';
 import { AssetType, ServiceName } from '../common/types';
 
+export const DisplayedAssets: Record<ServiceName, string[]> = {
+  kibana: ['index-pattern', 'visualization', 'search', 'dashboard'],
+  elasticsearch: ['index-template', 'ingest-pipeline', 'ilm-policy'],
+};
+
 export const AssetTitleMap: Record<AssetType, string> = {
-  config: 'Config',
   dashboard: 'Dashboard',
-  'index-pattern': 'Index Pattern',
+  'ilm-policy': 'ILM Policy',
   'ingest-pipeline': 'Ingest Pipeline',
+  'index-pattern': 'Index Pattern',
+  'index-template': 'Index Template',
   search: 'Saved Search',
-  'timelion-sheet': 'Timelion Sheet',
   visualization: 'Visualization',
 };
 
 export const ServiceTitleMap: Record<ServiceName, string> = {
   elasticsearch: 'Elasticsearch',
-  filebeat: 'Filebeat',
   kibana: 'Kibana',
-  metricbeat: 'Metricbeat',
 };
 
-export const AssetIcons: Record<AssetType, IconType> = {
-  config: 'advancedSettingsApp',
-  visualization: 'visualizeApp',
+export const AssetIcons: Record<string, IconType> = {
```

Comment:
I think it's important to keep the requirement that the keys are `AssetType`s, or some other type, vs any `string`. 

I'm making a PR to revert the `enum` to union types and hopefully that will make this easier.